pkg/manifests/config.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Cluster Monitoring Operator Authors
+/// Copyright 2018 The Cluster Monitoring Operator Authors
```

Comment:
The triple slash is unneeded i believe.

Reply:
I have no idea what happened here :D lets see if its my weird vim configuration, nice catch